### DISCUSSION TITLE
Should we add a disclaimer in the README

### DISCUSSION THREAD

**Discussion by AxeldeWater** [JunkFood02/Seal#165] [General]
I recently looked at the app Spotiflyer, which has the purpose of downloading Spotify songs. Spotiflyer has a disclaimer in their README:

Disclaimer
Downloading copyright songs may be illegal in your country. This tool is for educational purposes only and was created only to show how Music Platform's Apis like Spotify's API can be exploited to download music. Please support the artists by buying their music.

Wouldn't it be smart to have such disclaimer for Seal too? It's in most countries illegal to download copyrighted material from YouTube (most material on YouTube is copyrighted)

**Comment by JunkFood02** [comment]
~~I'm living in a country where even accessing YouTube is illegal and I'm quite ok with it~~

**Comment by AxeldeWater** [comment]
@JunkFood02 That's not the point. This project could be held accountable for pirating if YouTube ever decided to chase this project down, or if somebody would be caught pirating, using this project, they could point at this app saying "I didn't know it was illegal to use that app" and then the project gets taken down.

**Reply by BobbyESP** [reply]
Well, before that YouTube needs to take action with yt-dl (yt-dlp) first because Seal is just a GUI/Wrapper

**Reply by AxeldeWater** [reply]
I know, but I don't think we could point at yt-dlp if somebody points at this project (I think they do have a disclaimer), which would make us 'guilty'. I think we should add a disclaimer just in case.

**Reply by JunkFood02** [reply]
Yeah, I agree with that we're just a GUI for yt-dlp (youtube-dl). Since neither of them has a disclaimer and we're not mentioning any copyright infringed usages (while Spotiflyer does)

More about youtube-dl: https://github.blog/2020-11-16-standing-up-for-developers-youtube-dl-is-back/
